src/utils/datasets.ts

```diff
@@ -282,3 +286,34 @@ export const deleteDataset = async ({
 
   await apiClient.deleteDataset(id!);
 };
+
+const sleep = (ms: number) => new Promise((r) => setTimeout(r, ms));
+
+export const extendDataset = async (
```

Comment:
If this is the function that is exported publicy from the SDK, can you add documentation to it? We autogenerate SDK docs from docstrings in code, so all exported functions need to be well documented.